### PR TITLE
feat: Add parameter for CREATE LABEL

### PR DESCRIPTION
**Pull request by gtyun**
Compared with CREATE TABLE,
Add : unlogged, if not exists, with, tablespace
Not : temp, on commit, of type_name, callate, like, constraint,
     check, unique, primary key, exclude, deferrable

### CODE REVIEW

File: src/backend/parser/parse_utilcmd.c [hunk 1]
```diff
@@ -3049,9 +3049,6 @@ transformCreateLabelStmt(CreateLabelStmt *labelStmt, const char *queryString)
 							stmt->relation->relname)));
 	}
 
-	stmt->oncommit = ONCOMMIT_NOOP;
-	stmt->if_not_exists = false;
-
```
Comment (protodef) [inline src/backend/parser/parse_utilcmd.c]: Setting appropriate values to `options`, `oncommit`, `tablespacename`, and `if_not_exists` fields in `stmt` is missing. You should use `labelStmt` to set those values.

**Comment by gtyun** [comment]
Fix and add some query to regression ddl test.

File: src/include/nodes/parsenodes.h [hunk 2]
```diff
@@ -3109,6 +3109,10 @@ typedef struct CreateLabelStmt
 	LabelKind	labelKind;		/* LABEL_VERTEX or LABEL_EDGE */
 	RangeVar   *relation;		/* relation to create */
 	List	   *inhRelations;	/* relations to inherit from */
+	List	   *options;		/* options from WITH clause */
+	OnCommitAction oncommit;	/* what do we do at COMMIT? */
```
Comment (protodef) [inline src/include/nodes/parsenodes.h]: Based on the commit message you wrote, there is no need to keep this `oncommit` field in this struct (and all node functions for it).

File: src/test/regress/sql/cypher_ddl.sql [hunk 3]
```diff
@@ -15,6 +15,31 @@ set graph_path=mygraph,urgraph;
...
+CREATE VLABEL duple;
+CREATE VLABEL IF NOT EXISTS duple;
+DROP VLABEL duple;
+
+-- enable set tablespace
+CREATE VLABEL setspace tablespace pg_default;
+DROP VLABEL setspace;
+
+-- WITH
+CREATE VLABEL withv with oids;
```
Comment (protodef) [inline src/test/regress/sql/cypher_ddl.sql]: Do we need to support `WITH OIDS` and `WITHOUT OIDS` options? It seems that there is no reason to assign OIDs for each rows in labels. If we don't need it, there must be a check routine for user input options. `interpretOidsOption()` might help.

Comment (gtyun) [inline src/test/regress/sql/cypher_ddl.sql]: OID is substitue for primary key. So OID is not necessary to graph label. 
So I fixed the grammar, exchange OptWith to opt_reloptions.
User cannot query CREATE LABEL 'with oids, without oids'. And 'with (oids)' will not work anymore.

**Comment by protodef** [comment]
I've rebased and merged it.

**Pull request closed without merging**